app/lib/frontend/templates/views/pkg/info_box.dart

```diff
@@ -105,6 +109,23 @@ d.Node packageInfoBoxNode({
   ]);
 }
 
+d.Node _downloadsChart(WeeklyDownloadCounts wdc) {
+  final container = d.div(
+      classes: ['weekly-downloads-sparkline'],
+      id: '-weekly-downloads-sparkline',
+      attributes: {
+        'data-widget': 'weekly-sparkline',
+        'data-sparkline-points':
+            _encodeForWeeklySparkline(wdc.weeklyDownloads, wdc.newestDate),
+      });
+  return container;
+}
+
+String _encodeForWeeklySparkline(List<int> downloads, DateTime newestDate) {
+  final date = newestDate.toUtc().millisecondsSinceEpoch ~/ 1000;
+  return encodeIntsAsLittleEndianBase64String([date, ...downloads]);
+}
```

Comment:
You could just do:
```suggestion
String _encodeForWeeklySparkline(List<int> downloads, DateTime newestDate) {
  final date = newestDate.toUtc().millisecondsSinceEpoch ~/ 1000;
  final data = Uint32List.fromList([date, ...downloads]);
  return base64.encode(Uint8List.sublistView(data));
}
```
See: https://github.com/dart-lang/pub-dev/pull/8206/files#r1822801873

Reply:
Great, I didn't know `Uint8List.sublistView()` can translate any given typed data object to bytes. I'll update this in a follow up.